src/frontend/components/render-state/render-state.css

```diff
@@ -0,0 +1,56 @@
+.emitter {
+  position: relative;
+  display: flex;
+  width: 100%;
+  border-radius: 2px;
+  box-shadow: none;
+  -webkit-transition: all 0.4s linear;
```

Comment:
Prefixed rules are not necessary since we're only deploying in chrome, and as a stretch, WebKit browsers from this decade.